client/src/components/App.jsx

```diff
@@ -34,11 +34,12 @@ class App extends React.Component {
           currentProductID={this.state.currentProductID}/>
         <Reviews productId={Reviewsdata}/>
         <QuestionList
-          data={QuestionListdata}
-          currentProductID={this.state.currentProductID}/>
+          data={this.state.questionList.QuestionListdata.QuestionList}
+          currentProductID={this.state.currentProductID}
+          answerParser={this.state.questionList.QuestionListdata.answerParser}/>
```

Comment:
What is the answerParser property?

Reply:
It's sort of a temporary solution to not having a Client-Server-DB file structure, so I'm just directly separating out the answerParser (it's a function that takes all the answers from the answers object within a given question and returns them in an iterable array so I can map over them and render divs in my QuestionListEntry component). Sorry for the lack of explanation!